vcd/resource_vcd_catalog_item.go

```diff
@@ -137,7 +140,12 @@ func resourceVcdCatalogItemCreate(d *schema.ResourceData, meta interface{}) erro
 		return fmt.Errorf("error waiting for task to complete: %+v", err)
 	}
 
-	d.SetId(catalogName + ":" + itemName)
+	item, err := catalog.GetCatalogItemByName(itemName, true)
+	if err != nil {
+		return fmt.Errorf("error retrieving catalog item %s: %s", itemName, err)
+	}
+	// d.SetId(catalogName + ":" + itemName)
+	d.SetId(item.CatalogItem.ID)
```

Comment:
A note in changelog about changing ID format.

Reply:
done